definition_validator/validator.cc

```diff
@@ -278,9 +279,9 @@ class ValidateWalk {
 public:
     unique_ptr<ast::ClassDef> preTransformClassDef(core::Context ctx, unique_ptr<ast::ClassDef> classDef) {
         auto sym = classDef->symbol;
-        validateAbstract(ctx.state, sym);
-        validateTStructNotGrandparent(ctx.state, sym);
         auto singleton = sym.data(ctx)->lookupSingletonClass(ctx);
+        validateTStructNotGrandparent(ctx.state, sym);
```

Comment:
ooc: why did the order change?

Reply:
Just to line up the calls to validateAbstract one after the other so it looks nicer.